Source/Core/DolphinWX/Config/AdvancedConfigPane.cpp

```diff
@@ -45,10 +46,11 @@ void AdvancedConfigPane::InitializeGUI()
   m_custom_rtc_time_picker = new wxTimePickerCtrl(this, wxID_ANY);
 
   wxStaticText* const clock_override_description =
-      new wxStaticText(this, wxID_ANY, _("Higher values can make variable-framerate games "
-                                         "run at a higher framerate, at the expense of CPU. "
-                                         "Lower values can make variable-framerate games "
-                                         "run at a lower framerate, saving CPU.\n\n"
+      new wxStaticText(this, wxID_ANY, _("Adjusts the emulated CPU's clock rate.\n\n"
+                                         "Higher values may make variable-framerate games run "
+                                         "at a higher framerate, at the expense of performance. "
+                                         "Lower values may activate a game's internal "
+                                         “frameskip, potentially improving performance.\n\n"
```

Comment:
This must use `"` instead of `“`, otherwise compilation will fail.

I'm a bit worried that people won't understand that activating a game's internal frameskip means reducing the framerate and that they because of that will think that lowering this setting always is a great idea because it says it can improve the performance.